msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -47,11 +47,17 @@
 
 /**
  * <p>
- * This is the entry point for developer to create the public native applications and make API call to acquire tokens. MSAL {@link PublicClientApplication} provides three constructors allowing the client id to be set either via AndroidManifest.xml metadata or using constructor parameters.
+ * This is the entry point for developers to create the public native application and make API calls to acquire tokens.
+ * <p><b>Client ID:</b>  The clientID of your application is a unique identifier which can be obtained from the app portal.</p>
+ * <p><b>Authority:</b> A URL indicating a directory that MSAL can use to obtain tokens. In Azure AD
```

Comment:
right... in the code it might be coming back as a string, but it is meant to be a URL

---
In reply to: [114632311](https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/160#discussion_r114632311) [](ancestors = 114632311)